content/updates/quality/1-The-quality-score.md

```diff
@@ -0,0 +1,33 @@
+# Introducing our Quality score!
+
+## A new way of scoring...
+::: meta-data type="New" ee-only="false" link-to-doc="../articles/understand-data-quality.html#about-data-quality-insights"
+:::
+Measuring your product quality is now easier than ever thanks to the **quality score**, our new simplified way to score your product. It replaces the enrichment and consistency grades for an enhanced clarity.
+
+![The new quality score](../img/quality-score-in-pef.png)
+
+You can now find this score next to the completeness in the header of each product page, as well as in the product grid. Don't hesitate to use it as a filter to focus your attention on the products with low-quality scoring.
+
+## ...All your products...
+::: meta-data type="New" ee-only="false"
+:::
+You'll also notice that this score and the Data Quality Insights tab are now available for both your variant products and product models! Get specific quality scores for all your product variants and quickly identify the list of attributes you need to improve for a specific variant product. We also deliver a quality score at the product model level in which you will see at a glance on which level the attribute to improved is (product variant or model level).
+
+![Data Quality Insights on variant products](../img/dqi_variant_products.png)
+
+To enhance your productivity, you can also notice that, under the Data Quality Insights tab, if you have any improvement recommendations, you can simply click on your attribute to directly access it and improve it! 
```

Comment:
```suggestion
When you have improvement recommendations, you can simply click on your attribute to go directly to it and improve it under the Data Quality Insights tab. This will make your job easier and allow you to react to poor data quality faster.
```